ramls/schema/fixedField.json

```diff
@@ -0,0 +1,38 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "title": "fixedField",
+  "description": "The control field values",
+  "properties": {
+    "categoryCode": {
+      "type": "integer",
+      "description": "marc category"
+    },
+    "headerTypeCode": {
+      "type": "integer",
+      "description": "header type code"
+    },
+    "code": {
+      "type": "string",
+      "description": "marc tag"
+    },
+    "displayValue": {
+      "type": "string",
+      "description": "the tag value"
+    },
+    "sequenceNumber": {
+      "type": "integer",
+      "description": "order in the record"
+    },
+    "attributes": {
+      "type": "object",
+      "description": "attributes"
+    }
```

Comment:
Is there any structure to this field? This seems to defeat the purpose of the `additionalProperties` field. While there may be cases of fields that are just `object` in other FOLIO code, I am not aware of them. Can this field be bounded in some way or is it truly arbitrary? There are security concerns with respect to storing unstructured data.